config/management/src/lib.rs

```diff
@@ -309,17 +319,27 @@ pub mod tests {
     #[test]
     #[ignore]
     fn test_validator_config() {
-        let namespace = "validator_config";
-        let mut storage = default_storage(namespace.into());
-        initialize_storage(storage.as_mut());
+        let local_ns = "local_validator_config";
+        let remote_ns = "remote_validator_config";
+        let mut local = default_storage(local_ns.into());
+        initialize_storage(local.as_mut());
 
-        validator_config(
+        let mut remote = default_storage(remote_ns.into());
+        remote.reset_and_clear().unwrap();
+
+        let local_txn = validator_config(
```

Comment:
It doesn't look like we have any test that actually verifies the txn returned by validator_config(), right? It would be nice to have some simple assert_eq!() on various fields just to make sure nothing is really broken?